src/components/pages/deployments/deploymentsHome/flyouts/deploymentNew/deploymentNew.js

```diff
@@ -250,23 +277,47 @@ export class DeploymentNew extends LinkedComponent {
               }
             </FormGroup>
             <FormGroup className="new-deployment-formGroup">
-              <FormLabel isRequired="true">{t('deployments.flyouts.new.type')}</FormLabel>
+              <FormLabel isRequired="true">{t('deployments.flyouts.new.packageType')}</FormLabel>
               {
                 !completedSuccessfully &&
                 <FormControl
                   type="select"
                   className="long"
                   link={this.packageTypeLink}
                   onChange={this.onPackageTypeSelected}
-                  options={typeOptions}
-                  placeholder={t('deployments.flyouts.new.typePlaceHolder')}
+                  options={packageTypeSelectOptions}
+                  placeholder={t('deployments.flyouts.new.packageTypePlaceHolder')}
                   clearable={false}
                   searchable={false} />
               }
               {
                 completedSuccessfully && <FormLabel className="new-deployment-success-labels">{packageType}</FormLabel>
               }
             </FormGroup>
+            {configTypeEnabled &&
+              <FormGroup className="new-deployment-formGroup">
+                <FormLabel isRequired="true">{t('deployments.flyouts.new.configType')}</FormLabel>
+                {!completedSuccessfully &&
+                  <FormControl
+                    type="select"
+                    className="long"
+                    onChange={this.configTypeChange}
+                    link={this.configTypeLink}
+                    options={configTypeSelectOptions}
+                    placeholder={t('deployments.flyouts.new.configTypePlaceHolder')}
+                    clearable={false}
+                    searchable={false} />
+                }
+                {configTypesIsPending && <Indicator />}
+                {
+                  /** Displays an error message if one occurs while fetching configTypes. */
+                  configTypesError && <AjaxError className="new-deployment-flyout-error" t={t} error={configTypesError} />
+                }
+                {
```

Comment:
are these new lines necessary?

Reply:
Yes without these lines VS code color coding is  messed up for me. It showed everything after this line as comments even though they are not.